worker/include/RTC/RTCP/Packet.h

```diff
@@ -140,9 +137,9 @@ namespace RTC { namespace RTCP
 	}
 
 	inline
-	uint8_t* Packet::GetRaw()
+	const uint8_t* Packet::GetData()
 	{
-		return this->raw;
+		return (uint8_t*)this->header;
 	}
 }}
```

Comment:
The method is `const uint8_t*` but it returns a `uin8_t*), doesn't it cause a warning?